### PR TITLE
chore: Translations for the categories taxonomy

### PR DESCRIPTION
**Pull request by teolemon**
Translations for the categories taxonomy

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -63488,13 +64340,15 @@ ciqual_food_name:fr:Saucisse (aliment moyen)
...
 fr:Chair à saucisse
+it:Carne per salsicce
 agribalyse_food_code:en:30050
 ciqual_food_code:en:30050
 ciqual_food_name:en:Sausage meat, raw
 
 <en:Sausages
 en:Pork sausages
 fr:Saucisses de porc, Saucisses au porc
+it:Salsicce di maiale
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt:64351]:
```suggestion
it:Salsicce di suino
```

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -63779,6 +64635,7 @@ ciqual_food_name:fr:Saucisse de foie
 en:Pork liver sausages
 de:Schweineleberwürste, Schweineleberwurst
 fr:Saucisses de foie de porc
+it:Salsicce di fegato di maiale
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt:64638]:
```suggestion
it:Salsicce di fegato di suino
```

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -74137,6 +75148,7 @@ fr:Pâtes à tartiner aux noisettes et au cacao, pâtes à tartiner aux noisette
 hu:Kenhető csokoládé és mogyoró
 it:Crema con nocciole e cacao
 nl:Choco-hazelnootpastas
+pt:Cacau e avelãs para barrar
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt:75151]:
```suggestion
pt:Creme para barrar de avelã e cacau
```

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -74122,6 +75132,7 @@ fr:Pâtes à tartiner aux noisettes, pâte à tartiner aux noisettes
 hu:Kenhető mogyoró
 it:Crema con nocciole, crema alle nocciole
 nl:Hazelnootpastas
+pt:Avelãs para barrar
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt:75135]:
```suggestion
pt:Creme para barrar de avelã 
```

File: taxonomies/categories.txt [hunk 5]
```diff
@@ -74100,7 +75108,9 @@ he:ממרחי שוקולד
 hu:Kenhető csokoládé
 it:Crema con cacao
 nl:Chocopastas, Chocoladepastas
+pt:Chocolates para barrar
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt:75111]:
```suggestion
pt:Creme para barrar de chocolate
```

**Review by moon-rabbitOFF — CHANGES_REQUESTED** [review]

**Comment by moon-rabbitOFF** [comment]
I think that would be good that someone else with a knowledge of the taxonomy proofreads again this massive update. There are numerous languages that I couldn't review

File: taxonomies/categories.txt [hunk 6]
```diff
@@ -26673,6 +26880,7 @@ protected_name_type:en: pgi
 <en:Chocolates
 en:Assorted chocolates
 de:Verschiedene Pralinen, Pralinensortimente
+es:Surtido de chocolates, Surtido de chocolatinas, Surtido de bombones
```
[the remark this answers is not in the document]
Comment (benbenben2) [reply taxonomies/categories.txt:26883]: I would keep " Surtido de chocolatinas, Surtido de bombones" as it looks like synonyms. What do you think @moon-rabbitOFF ?

File: taxonomies/categories.txt [hunk 7]
```diff
@@ -40895,12 +41422,14 @@ pt:Gorduras
 ru:Жиры и масла, Масла и жиры, Масла, Масло
 sv:Fett, Matfett
 th:ไขมัน
+tr:Yağ (katı)
```
Comment (benbenben2) [inline taxonomies/categories.txt:41425]:
```suggestion
tr:Yağ
```

File: taxonomies/categories.txt [hunk 8]
```diff
@@ -41112,6 +41644,7 @@ pl:Tłuszcze roślinne
 pt:Gorduras vegetals
 ru:Растительные жиры, Жиры растительные
 th:ไขมันจากพืช
+tr:Bitkisel yağ (katı)
```
Comment (benbenben2) [inline taxonomies/categories.txt:41647]:
```suggestion
tr:Bitkisel yağ
```

File: taxonomies/categories.txt [hunk 9]
```diff
@@ -41353,6 +41889,7 @@ pt:Olios vegetais, olios baseados om plantas
 ru:Масло растительное, Растительное масло
 sv:Vegetabiliska oljor, vegetabilisk olja
 th:น้ำมันพืช
+tr:Bitkisel yağlar (sıvı)
```
Comment (benbenben2) [inline taxonomies/categories.txt:41892]:
```suggestion
tr:Bitkisel yağlar
```

**Comment by benbenben2** [comment]
@moon-rabbitOFF , @teolemon I made a local merge with main to resolve all conflicts. Now is the chance to close this pull request!

**Comment by benbenben2** [comment]
@moon-rabbitOFF , @teolemon, @stephanegigandet,  I made a local merge with main again to resolve all conflicts. Now is the chance to close this pull request!

**Comment by stephanegigandet** [comment]
> @moon-rabbitOFF , @teolemon, @stephanegigandet, I made a local merge with main again to resolve all conflicts. Now is the chance to close this pull request!

Wonderful, thank you @benbenben2 !

**Pull request merged**